Makefile

```diff
@@ -784,6 +814,27 @@ else
 	@$(MV) $(@F) $(INSTALL_LIBDIR)/
 endif
 
+# --- Install-netlibs-test dir ---
+
+install-netlib-tests:
+ifeq ($(FLA_ENABLE_LEGACY_LAPACK),no)
+ifeq ($(FLA_ENABLE_LAPACK2FLAME),yes)
```

Comment:
ONLY build the new test if not running --enable-legacy-lapack and either lapack2flame or lapack2flash have been enabled.